driver/web/adapter.go

```diff
@@ -45,7 +48,7 @@ type Adapter struct {
 
 // @title Rokwire Notifications Building Block API
```

Comment:
All this comments were used by the old Swagger implememntation. We can remove them now.